src/dataprocessor/xmldataprocessor.js

```diff
@@ -89,7 +89,7 @@ export default class XmlDataProcessor {
 		const domFragment = this._toDom( data );
 
 		// Convert DOM DocumentFragment to view DocumentFragment.
-		return this._domConverter.domToView( domFragment );
+		return this._domConverter.domToView( domFragment, { keepOriginalCase: true } );
```

Comment:
This change was done because XmlDP is used in the dev utils to parse textual representation of the model. We use elements like <listItem> and it should be converted to listItem in the view and then in the model. Not the cleanest change, but perhaps it makes sense that XmlDP works this way (comparing to HtmlDP's lowercasing everything).